app/api/v1/admin/categories.rb

```diff
@@ -0,0 +1,77 @@
+module V1
+  module Admin
+    class Categories < API
+      resources :categories do
+        desc 'Return all categories'
+        get do
+          categories = Category.all
+          present categories, with: Entities::Category
+        end
+
+        desc 'Return a specific category'
+        route_param :id do
+          get do
+            category = Category.find(params[:id])
+            present category, with: Entities::Category
+          rescue ActiveRecord::RecordNotFound
+            error!({ error_code: 404, error_message: 'Category not found' })
```

Comment:
We decided to separate this kind of logic in some general class/module and do it [in the scope of this ticket](https://trello.com/c/wGRbj1hx/70-add-error-handling-for-crud-operations)

Reply:
That's why I'd recommend removing it from the current PR